Dockerfile

```diff
@@ -1,5 +1,19 @@
 FROM node:current-alpine
 
+ARG GIT_COMMIT
+ARG WORKBENCH_CLIENT_VERSION="latest"
```

Comment:
This should not default to `latest` - latest is a tag (alias) for a version, not a version itself